src/window/Window.cpp

```diff
@@ -47,6 +47,7 @@ Window::~Window() {
 void Window::Init() {
     bool steamDeckGameMode = false;
     bool androidGameMode = false;
+    bool iosGameMode = false;
```

Comment:
do we use these separately at all? seems like we could probably just use a single "gameMode" variable and set it appropriately based on platform